examples/graphlearn_for_pytorch/distributed/README.md

```diff
@@ -0,0 +1,125 @@
+# Distributed (multi nodes) examples.
+
+This example shows how to leverage [GraphLearn-for-PyTorch(GLT)](https://github.com/alibaba/graphlearn-for-pytorch) to train PyG models in distributed scenarios with GPUs. The dataset in this example is `ogbn-products` from the [Open Graph Benchmark](https://ogb.stanford.edu/) and you can also train on `ogbn-papers100M` with minor modifications of the configuration file.
+
+To run this distributed example, you can choose to direct run the example or use our [script](launch.py) following the guidance below.
+
+## Reference test environment and dependency packages
+docker image: pytorch/pytorch:1.13.0-cuda11.6-cudnn8-devel
+```
+pip install torch==1.13.0+cu117 torchvision==0.14.0+cu117 torchaudio==0.13.0 --extra-index-url https://download.pytorch.org/whl/cu117
+pip install torch_geometric
+pip install --no-index  torch_scatter torch_sparse -f https://data.pyg.org/whl/torch-1.13.0+cu117.html
+pip install graphlearn-torch
+```
+
+## Option 1: Directly run the example
+### Step 1: Prepare and partition data.
+Here we use ogbn_products and partition it into 2 partitions.
+```
+python partition_ogbn_dataset.py --dataset=ogbn-products --root_dir=../../../data/ogbn-products --num_partitions=2
+```
+
+### Step 2: Run the example of distributed training
+2 nodes each with 2 GPUs
+```
+# node 0:
+CUDA_VISIBLE_DEVICES=0,1 python dist_train_sage_supervised.py \
+  --num_nodes=2 --node_rank=0 --master_addr=localhost \
+  --dataset=ogbn-products --dataset_root_dir=../../../data/ogbn-products \
+  --in_channel=100 --out_channel=47
+
+# node 1:
+CUDA_VISIBLE_DEVICES=2,3 python dist_train_sage_supervised.py \
+  --num_nodes=2 --node_rank=1 --master_addr=localhost \
+  --dataset=ogbn-products --dataset_root_dir=../../../data/ogbn-products \
+  --in_channel=100 --out_channel=47
```

Comment:
you should better add papers100M config here.

Reply:
added.